### PR TITLE
Support for environment variables

### PR DESCRIPTION
**Pull request by blakewenloe**
### Motivation

Fix for [#134](https://github.com/Kentico/cloud-sample-app-react/issues/134) by supporting environment variables in Client.js

### Checklist

- [ ] Code follows coding conventions held in this repo
- [ ] Automated tests have been added
- [ ] Tests are passing
- [ ] Docs have been updated (if applicable)
- [ ] Temporary settings (e.g. variables used during development and testing) have been reverted to defaults

### How to test

Test by following the updated README, and configuring your environment variables.

### CODE REVIEW

**Comment by blakewenloe** [comment]
Apparently `dotenv` isn't required for accessing environment variables in React?

**Comment by blakewenloe** [comment]
> Could you also change the lists to start with just one space to fulfill the default indentation
> 
> * https://github.com/DavidAnson/markdownlint/blob/v0.15.0/doc/Rules.md#md030
> 
> Thanks to notifying about the [out of the box support ](of the environment variables).
> 
> Sorry for the long delay - one of the CI checks didn't finish so that I didn't get any notification.

No worries! Let me know if you need any other changes

File: README.md [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
-
 ## Application setup
 
-1. Install the latest version of NodeJS and npm. You can download both at <https://nodejs.org/en/download/>.
-2. Clone the sample application repository.
-3. Navigate to the root folder of the application in the command line.
-4. Type `npm install` to install required npm packages.
-5. Type `npm start` to start a development server.
-6. The application opens in your browser at <http://localhost:3000>.
+1.  Install the latest version of NodeJS and npm. You can download both at <https://nodejs.org/en/download/>.
```
Comment (Simply007) [inline README.md]: Could you also change the lists to start with just one space to fulfill the default indentation there?

https://github.com/DavidAnson/markdownlint/blob/v0.15.0/doc/Rules.md#md030

File: README.md [hunk 2]
```diff
@@ -26,18 +26,18 @@ Alternatively, you can connect your project manually as per the chapter below.
...
- 
-1. In Kentico Cloud, choose Project settings from the app menu.
-2. Under Development, choose API keys.
-3. Copy your Project ID
-4. Open the [`src\Client.js`](/src/Client.js) file in the sample application folder.
-5. Find the `projectId` constant.
-6. Change the values of the constant using the Project ID key you copied.
-7. Save the file.
+
+1.  In Kentico Cloud, choose Project settings from the app menu.
```
Comment (Simply007) [inline README.md]: Could you also change the lists to start with just one space to fulfill the default indentation there?

https://github.com/DavidAnson/markdownlint/blob/v0.15.0/doc/Rules.md#md030

File: README.md [hunk 3]
```diff
@@ -48,30 +48,29 @@ If you already have a Kentico Cloud account and you want to connect the sample a
...
 To preview content in the sample application, follow these steps:
 
-1. In Kentico Cloud, choose Project settings from the app menu.
-2. Under Development, choose API keys.
-3. Copy your Project ID and Preview API key.
-4. Open the `src\Client.js` file in the sample application folder.
-5. Find the `projectId` and `previewApiKey` constants.
-6. Change the values of the constants using the Project ID and Preview API key you copied.
-7. Save the file.
+1.  In Kentico Cloud, choose Project settings from the app menu.
```
Comment (Simply007) [inline README.md]: Could you also change the lists to start with just one space to fulfill the default indentation there?

https://github.com/DavidAnson/markdownlint/blob/v0.15.0/doc/Rules.md#md030

File: README.md [hunk 4]
```diff
@@ -48,30 +48,29 @@ If you already have a Kentico Cloud account and you want to connect the sample a
...
+6.  Save and rename the file `.env`.
 
 When you now run the application, you will see all project content including the unpublished version of content items.
 
 ## Content administration
 
-1. Navigate to <https://app.kenticocloud.com> in your browser.
-2. Sign in with your credentials.
-3. Manage content in the content administration interface of your sample project.
+1.  Navigate to <https://app.kenticocloud.com> in your browser.
```
Comment (Simply007) [inline README.md]: Could you also change the lists to start with just one space to fulfill the default indentation there?

https://github.com/DavidAnson/markdownlint/blob/v0.15.0/doc/Rules.md#md030

File: README.md (hunk 1 above)
Comment (blakewenloe) [reply README.md]: Oh whoops

**Comment by Simply007** [comment]
fixes #136

**Comment by Simply007** [comment]
@blakewenloe - could you also [request assigning to Kentico GitHub Global collaborators](https://kentico.github.io/join), then it will be possible to assign you as an [issue](https://github.com/Kentico/cloud-sample-app-react/issues/134) fixer.

**Comment by blakewenloe** [comment]
@Simply007 Done!

**Comment by Simply007** [comment]
If you want to take a look at the news we are providing to our Global Collaborators feel free to check [Global Collaborators Flash](https://github.com/orgs/Kentico/teams/global-collaborators).

We also have a [public backlog of javascript enhancements](https://github.com/orgs/Kentico/projects/4) if you are interested!

**Comment by blakewenloe** [comment]
Oh nice! I'm checking it out now

**Pull request merged**